ios/Handlers/ResourcesHandler.swift

```diff
@@ -1,137 +1,82 @@
 protocol BridgeToResourceHandlerProtocol: AnyObject {
-    func authenticateDevice(_ scopes:[String], _ completion: @escaping (Bool, Error?) -> Void)
-    func authenticateImplicitly(_ profile: ONGUserProfile, scopes: [String], _ completion: @escaping (Bool, Error?) -> Void)
     func resourceRequest(_ type: ResourceRequestType, _ details: NSDictionary, _ completion: @escaping (String?, Error?) -> Void)
 }
 
 enum ResourceRequestType: String {
-    case User
-    case ImplicitUser
-    case Anonymous
+    case user
+    case implicitUser
+    case anonymous
+    
+    init(rawValue: String) {
+        switch rawValue {
+        case "User": self = .user
+        case "ImplicitUser": self = .implicitUser
+        case "Anonymous": self = .anonymous
+        default: self = .anonymous
+        }
+    }
 }
 
 class ResourceHandler: BridgeToResourceHandlerProtocol {
-    func authenticateDevice(_ scopes:[String], _ completion: @escaping (Bool, Error?) -> Void) {
-        ONGDeviceClient.sharedInstance().authenticateDevice(scopes) { success, error in
-            if let error = error {
-                completion(success, error)
-            } else {
-                completion(success, nil)
-            }
-        }
+    private var deviceClient: DeviceClient {
+        return SharedDeviceClient.instance
```

Comment:
If you don't "compute" this value somehow (it is always `SharedDeviceClient.instance`) you can have just 
```
private let deviceClient = SharedDeviceClient.instance